default/python/AI/freeorion_tools/_freeorion_tools.py

```diff
@@ -347,3 +348,22 @@ def wrapper(*args, **kwargs):
 
         return wrapper
     return decorator
+
+
+def object_is_in_ion_storm(object_id):
+    # FIXME: This isn't working if we can not detect the ion storm
```

Comment:
Although some warning  of uncertainty is suitable here, I think the situation is not as dire as the comment currently describes.  ```SetEmpireFieldVisibilitiesFromRanges()``` (in universe.cpp) grants Partial Visibility of a field if any of the empire's detection range simply touches the perimeter of the field, no need to directly see its center.

Perhaps also add a TODO about predicting field trajectories and size-- with a bit of work the AI should be able to make nearly perfect projections of such after a single sighting.